package/score-worker/src/main/java/io/cloudslang/schema/WorkerBeanDefinitionParser.java

```diff
@@ -135,26 +139,55 @@ private void registerSpecialBeans(Element element, ParserContext parserContext)
 		registerWorkerVersionService(element, parserContext);
 		registerSequentialExecution(element, parserContext);
 		registerRobotAvailabilityService(element, parserContext);
+		registerExecutionPreconditionService(element, parserContext);
+		registerExecutionPostconditionService(element, parserContext);
 	}
 
 	private void registerSequentialExecution(Element element, ParserContext parserContext){
 		String registerSequentialExecutionService = element.getAttribute("registerSequentialExecutionService");
-		if(!Boolean.FALSE.toString().equals(registerSequentialExecutionService)){
-			new BeanRegistrator(parserContext).CLASS(DefaultSequentialExecutionServiceImpl.class).register();
+		if(!FALSE.toString().equals(registerSequentialExecutionService)){
```

Comment:
Indentation "){"
if registersequentialexecutionservice can not be null go with non yodatalk (check line 149)